library/built-in/_prefabs/prefab_latent_v3.ts

```diff
@@ -14,7 +14,7 @@ export const ui_latent_v3 = () => {
                 form.group({
                     collapsible: false,
                     label: false,
-                    items: { batchSize, size: form.size({ label: false }) },
+                    items: { batchSize, size: form.size({ label: false, default: { modelType: 'SDXL 1024' } }) },
```

Comment:
isn't that a bit too dangerous by default ? most people still use 1.5, no ?

Reply:
Oh whoops, it was done for a screenshot and I forgot.

Reply:
no prob, merged it already, but won't merge dev very very soon, so we can revert that line soon